File: src/Utility.java
public class Utility {

    public static int countChar(String target, String find) {
        String t = target;
        int count = 0;
        while (t.contains(find)) {
            t = target.replaceFirst(find, "");
            count++;
        }
        Log.i("E");

        return count;
    }

}

File: src/Utility.kt
class Utility {
    companion object {
        fun countChar(target: String, find: String): Int {
            var t = target;
            var count: Int = 0;
            while (t.contains(find)) {
                t = target.replaceFirst(find, "");
                count++;
            }
            Log.i("E");

            return count;
        }
    }
}